packages/forms-web-app/src/controllers/final-comment/check-your-answers.js

```diff
@@ -0,0 +1,49 @@
+const { VIEW } = require('../../lib/views');
+const { storeTextAsDocument } = require('../../services/pdf.service');
+const { documentTypes } = require('@pins/common');
+
+exports.getCheckYourAnswers = async (req, res) => {
+	res.render(VIEW.FINAL_COMMENT.CHECK_YOUR_ANSWERS, {
+		hasComments: req.session?.finalComment?.hasComments,
+		finalComments: req.session?.finalComment
```

Comment:
call the key `finalComment` for consistency? (would need to update nunjucks too)